packages/subgraph/schema.graphql

```diff
@@ -536,7 +537,8 @@ type AddresslistVotingProposal implements IProposal @entity {
   startDate: BigInt!
   endDate: BigInt!
   creationBlockNumber: BigInt!
-  potentiallyExecutable: Boolean!
+  approvalReached: Boolean!
+isSignaling: Boolean!
```

Comment:
```suggestion
  isSignaling: Boolean!
```

Reply:
Fixed in 42ae70adc0882ca90e4b6a45c6a0d7bb1df1ebab